### DISCUSSION TITLE
Comment <https://searx.info/>

### DISCUSSION THREAD

**Discussion by MPeti1** [searx/searx#3030] [Public Searx instances: feedback]
Today I wanted to use searx.info, which I was using for some time already.
When the page loaded, though, all I was seeing is a message in the top left that reads "setting cookie...". Since I mostly have JS disabled on pages that don't need it to work, it was disabled on this page too.

![Screenshot from 2021-10-22 18-37-00](https://user-images.githubusercontent.com/9215491/138491625-90fe3bb8-7282-49ff-b451-40ce8038144f.png)

I've searched around if I can find out what happened, but wasn't able to find much, other than a comment from 2 years ago in commit [b2e283c](https://github.com/searx/searx-instances/commit/b2e283c) (which was removed 4 months ago in commit [d35098b](https://github.com/searx/searx-instances/commit/d35098b)), which increased my suspicion about what might have happened.

After enabling JS on the page, it starts working as one would expect, even if you disable JS later. I think the reason might be that one of the scripts have already set a cookie now, without which the page won't load.
Probably not very useful, but if JS is enabled, a cookie with the BCP name and a value of a 32 character long hexadecimal string is set.

An other thing to note is that the instance does not currently appear in the searx.space list.

I do not intend to say that that it should be removed from the list, just wanted to let you know (from the commit I mentioned it seemed you deal with things like this), and also wanted to see if anyone else has noticed it.

Oh and if you don't know much about this, feel free to close the issue.
Also, I think things like this might fit the discussion section better, did you consider enabling it for the repository?

**Comment by unixfox** [comment]
searx.info is not really removed, searx.space can't scan it as it can't pass the "cookie" test, see "Offline & error" section on https://searx.space/.

So it is not included in the main page anymore as searx.space can't verify that searx is actually running on the instance.

With the huge amount of SearX instances available, it's not really an issue to find other good SearX instances that would suit your needs.

I'll anyway add a comment in the searx instances configuration file (done in https://github.com/searx/searx-instances/commit/909940c13d0b5460b5a1f74f327fb2ecd86ed340).
